app/models/place.rb

```diff
@@ -4,6 +4,25 @@ class Place
   attr_accessor :type, :properties, :geometry
 
   def as_json(options = {})
-    { label: properties['objektgruppe'] == 'Gemeindeteil' ? properties['_title_'].split(', ')[-1] : properties['abkuerzung'].blank? ? properties['_title_'].split(', ')[-1] : properties['_title_'].split(', ')[-1] + ' (' + properties['abkuerzung'] + ')', bbox: geometry['type'] == 'Point' ? [geometry['coordinates'][0],geometry['coordinates'][1],geometry['coordinates'][0],geometry['coordinates'][1]] : [geometry['coordinates'][0][0][0],geometry['coordinates'][0][0][1],geometry['coordinates'][0][2][0],geometry['coordinates'][0][2][1]], transform_bbox: geometry['transform_bbox'], feature_id: properties['feature_id'] }
+    { label: label(properties), bbox: bbox(geometry), transform_bbox: geometry['transform_bbox'], feature_id: properties['feature_id'] }
+  end
+
+  private
+  
+  def label(properties)
+    title = properties['_title_']
+    title_without_community = title.split(', ')[-1]
+    case properties['objektgruppe']
+    when 'Gemeinde'
+      title
+    when 'Gemeindeteil'
+      title_without_community
+    else
+      properties['abkuerzung'].blank? ? title_without_community : title_without_community + ' (' + properties['abkuerzung'] + ')'
```

Comment:
```suggestion
      properties['abkuerzung'].blank? ? title_without_community : "#{title_without_community} (#{properties['abkuerzung']})"
```
Stringeinbettung statt Verkettung